index.html

```diff
@@ -2838,64 +2838,66 @@ <h3>Existing Device Described with a Thing
+                in certain application scenarios.
+            </p>
+            <p> A <a>Servient</a> in the role of a <a>Consumer</a> obtains the TD of a <a>Thing</a> 
+                using a discovery mechanism implemented in the individual deployment scenarios
+                (such as a management system such as <a>Thing Directory</a>, discovery protocol, 
+                 static assignment, etc. as stated above)
+                and creates a <a>Consumed Thing</a> based on the obtained TD. 
                 Internal system functions of a device such as attached sensors and actuators
                 can also optionally be represented as <a>Consumed Thing</a> abstractions.
             </p>
-            <p> The functions supported by the 
-                <a>Exposed Thing</a> and <a>Consumed Thing</a> abstractions
-                are provided internally to the <a>Thing</a>'s behavior implementation
-                via a programming language interface.
-                In the <a>WoT Scripting API</a>, these abstractions are represented by objects.
-                The behavior implementation (that is, the application logic)
-                running in the <a>Thing</a> can then
-                achieve its functions, such as appropriately responding to
-                interactions, by using this interface.
-            </p>
-            <p> A <a>Thing</a> may or may not be a device associated with physical
-                sensors and actuators; 
-                it may also be a service running in a gateway 
-                or in the cloud.  
-                Likewise a <a>Consumer</a> may or may not be 
-                an application service running on a gateway or cloud, it
-                may in some cases be necessary to implement <a>Consumer</a>
-                functionality on edge devices.  
-                In some cases a single <a>Thing</a>
-                simultaneously performs both roles using a single <a>WoT Runtime</a>,
-                in which case we call it a <a>Servient</a>.
+            <p>The functions supported by the <a>Consumed Thing</a> are provided internally 
+               to the <a>Consumer</a>'s behavior implementation through a programming language 
+               interface. In the <a>WoT Scripting API</a>, <a>Consumed Things</a> 
+               are represented by objects.
+               The behavior implementation (that is, the application logic) running in 
+               a <a>Thing</a> can engage in interactions with <a>Consumers</a> by using 
+               the programming language interface provided by the <a>Exposed Thing</a>.
+            </p>
+            <p> A <a>Thing</a> may not necessarily represent a physical device. 
+                <a>Things</a> can also represent a collection of devices, or virtual services 
+                running in a gateway or in the cloud.
+                Likewise, a <a>Consumer</a> may or may not represent 
```

Comment:
What does it represent, if it does not represent an application service?

Reply:
A Consumer can be implemented on an edge device, in a gateway or in the cloud.

Reply:
Thank you for incorporating those changes on CR-candidate-3 branch!